.github/workflows/sims.yml

```diff
@@ -63,9 +63,9 @@ jobs:
         with:
           go-version: "1.23"
           check-latest: true
-      - name: test-sim-nondeterminism
+      - name: test-sim-nondeterminism-streaming
```

Comment:
what is the difference here and why was this updated?

Reply:
This is the exact same test as before: `TestAppStateDeterminism`, but with the steaming params set. None of the sims tests in CI covers streaming. IMHO it makes sense to have 1 test covering this. It does not make sense to run `TestAppStateDeterminism` twice.

Reply:
yeah that makes sense.  This test is a superset of what we care about